### PR TITLE
[Merged by Bors] - chore: remove some unused variables

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Normed/Group/Seminorm.lean [hunk 1]
```diff
@@ -381,7 +381,7 @@ end GroupSeminorm
 see that `SMul R ℝ` should be fixed because `ℝ` is fixed. -/
 namespace AddGroupSeminorm
 
-variable [AddGroup E] [SMul R ℝ] [SMul R ℝ≥0] [IsScalarTower R ℝ≥0 ℝ] (p : AddGroupSeminorm E)
+variable [AddGroup E] [SMul R ℝ] [SMul R ℝ≥0] [IsScalarTower R ℝ≥0 ℝ]
```
Comment (grunweg) [inline Mathlib/Analysis/Normed/Group/Seminorm.lean:384]: Nice!

File: Mathlib/LinearAlgebra/BilinearMap.lean [hunk 2]
```diff
@@ -37,16 +37,16 @@ variable {R : Type*} [Semiring R] {S : Type*} [Semiring S]
...
 variable {M₂ : Type*} {N₂ : Type*} {P₂ : Type*}
-variable {Nₗ : Type*} {Pₗ : Type*}
-variable {M' : Type*} {N' : Type*} {P' : Type*}
+variable {Pₗ : Type*}
+variable {M' : Type*} {P' : Type*}
 variable [AddCommMonoid M] [AddCommMonoid N] [AddCommMonoid P]
 variable [AddCommMonoid M₂] [AddCommMonoid N₂] [AddCommMonoid P₂]
-variable [AddCommMonoid Nₗ] [AddCommMonoid Pₗ]
-variable [AddCommGroup M'] [AddCommGroup N'] [AddCommGroup P']
+variable [AddCommMonoid Pₗ]
```
Comment (grunweg) [inline Mathlib/LinearAlgebra/BilinearMap.lean:44]: optional: combine this with the previous line?

File: Mathlib/LinearAlgebra/BilinearMap.lean [hunk 3]
```diff
@@ -376,14 +376,13 @@ end CommSemiring
 
 section CommRing
 
-variable {R R₂ S S₂ M N P : Type*}
-variable {Mₗ Nₗ Pₗ : Type*}
-variable [CommRing R] [CommRing S] [CommRing R₂] [CommRing S₂]
+variable {R M : Type*}
+variable [CommRing R]
```
Comment (grunweg) [inline Mathlib/LinearAlgebra/BilinearMap.lean:380]: optional: combine with the previous line?

File: Mathlib/LinearAlgebra/BilinearMap.lean [hunk 4]
```diff
@@ -376,14 +376,13 @@ end CommSemiring
...
 
 section AddCommGroup
 
-variable [AddCommGroup M] [AddCommGroup N] [AddCommGroup P]
-variable [Module R M] [Module S N] [Module R₂ P] [Module S₂ P]
+variable [AddCommGroup M]
+variable [Module R M]
```
Comment (grunweg) [inline Mathlib/LinearAlgebra/BilinearMap.lean:385]: Here also?

File: Mathlib/RingTheory/NonUnitalSubring/Basic.lean [hunk 5]
```diff
@@ -1006,10 +994,8 @@ end NonUnitalSubring
 
 namespace RingEquiv
 
-variable {F : Type w} {R : Type u} {S : Type v} {T : Type*}
-  [NonUnitalRing R] [NonUnitalRing S] [NonUnitalRing T]
-  [FunLike F R S] [NonUnitalRingHomClass F R S]
-  (g : S →ₙ+* T) (f : R →ₙ+* S)
+variable {R : Type u} {S : Type v}
```
Comment (grunweg) [inline Mathlib/RingTheory/NonUnitalSubring/Basic.lean:997]: Reflow to fit on one line?

**Review by grunweg — COMMENTED** [review]
Just three nits, which you may argue are pre-existing. Thanks!
maintainer merge

**Comment by sgouezel** [comment]
bors d+
Thanks!

File: Mathlib/LinearAlgebra/BilinearMap.lean (hunk 2 above)
Comment (adomani) [reply Mathlib/LinearAlgebra/BilinearMap.lean]: Done.

File: Mathlib/LinearAlgebra/BilinearMap.lean (hunk 3 above)
Comment (adomani) [reply Mathlib/LinearAlgebra/BilinearMap.lean]: Also here.

File: Mathlib/LinearAlgebra/BilinearMap.lean (hunk 4 above)
Comment (adomani) [reply Mathlib/LinearAlgebra/BilinearMap.lean]: And here indeed!

File: Mathlib/RingTheory/NonUnitalSubring/Basic.lean (hunk 5 above)
Comment (adomani) [reply Mathlib/RingTheory/NonUnitalSubring/Basic.lean]: Reflowed.

**Review by adomani — COMMENTED** [review]
Thanks for the review!

**Comment by adomani** [comment]
bors merge

**Pull request closed without merging**